src/containers/EventForm/components/EventForm/components/FirstStep/function.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ArtworksRepository,
+  PeopleRepository,
+  LocationsRepository,
+} from 'hackoss';
+import { FirebaseApp } from 'utils/firebase';
+
+async function getArtworksBase() {
+  const pplRepo = new PeopleRepository(FirebaseApp);
+  const awRepo = new ArtworksRepository(FirebaseApp, pplRepo);
+  const artwork = await awRepo.getArtworks();
+  return artwork;
+}
+async function getLocationsBase() {
+  const locRepo = new LocationsRepository(FirebaseApp);
+  const loc = await locRepo.getLocations();
+  return loc;
+}
+export async function getArtworks() {
+  const data = await getArtworksBase();
+  return data.length > 0 ? data.map(item => item.title) : ['Nothing created'];
```

Comment:
Is the `Nothing created` a placeholder? A little unsettling that the placeholder, a UI concern, is placed here in the function. May not be sure if it's a react convention.

Reply:
Yea.. It should be a placeholder. This is actually a ui bug that should be fixed soon.

Reply:
The correct fix should be disable the select components. But I currently have no implementation on this.

Reply:
Current solution will just show no select content. cuz Nothing created will actually create a item that can be selected, which turns out to be a potential error.